modules/Galleries/GalleriesList.module.scss

```diff
@@ -1,5 +1,5 @@
 .container {
-    padding: $spacing-8 0 0;
+    padding: 0;
```

Comment:
You can remove this class entirely, since the block will not have padding set either way.